### DISCUSSION TITLE
<Strudent developer  pack>

### DISCUSSION THREAD

**Discussion by ETV-Code** [community/community#103011] [GitHub Education]
### Select Topic Area

Question

### Body

Hello guys, I'm trying to use the student developer package and I can't, it says that I had already had a request approved but it won't let me log in to github copilot, I tried to send the request again but it tells me that I can't send the same certificate. What I can do?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ETV-Code ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
